packages/agent/conf/examples/oid4vci_metadata/knb_kvk.json

```diff
@@ -0,0 +1,168 @@
+{
+  "correlationId": "https://ssi.dutchblockchaincoalition.org/issuer",
```

Comment:
kvk.demo.sphereon.com everywhere for this file

Reply:
Fixed! Should I change the DBCConferenceAttendee in the credential type to something more meaningful?

Reply:
Yes of course,  as that denotes the context. This obviously cannot reuse the context we created for guest credentials